src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -45,10 +45,21 @@ def __init__(self, doc_name, app):
         """
         self._doc_name = doc_name
         self.doc_path = f"{self._doc_name}.html"
+        self.env = app.env
         self.doc_title = app.env.titles[self._doc_name].astext()
         self._doc_tree = app.env.get_doctree(self._doc_name)
         self.sections = []
 
+    def get_title_breadcrumbs(self):
+        """Generate title breadcrumbs from the document name."""
+        docs_parts = self._doc_name.split("/")[:-1]
```

Comment:
Could we document what is the last element and why are we excluding it? Just so we do not forget.